### DISCUSSION TITLE
Is it possible to change table formatting for PowerPoint documents in Quarto

### DISCUSSION THREAD

**Discussion by MrDuck01** [quarto-dev/quarto-cli#7840] [Q&A]
### Description

Hi all, I've got a quarto presentation putting content into a PowerPoint template and for plots, titles, etc everything is working as planned, but for tables, it defaults to a standardised font, colours etc and I dont seem to be able to amend it?  I've tried outputting the table via gt and kable but both generate errors, is there a better way?

**Comment by mcanouil** [comment]
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

![A placeholder image](https://placehold.co/600x400.png)

The end.
````
`````

**Comment by cderv** [comment]
> it defaults to a standardised font, colours etc and I dont seem to be able to amend it?

With Powerpoint, customisation goes through a Template (https://quarto.org/docs/presentations/powerpoint.html#powerpoint-templates) 
This is a Pandoc feature: https://pandoc.org/MANUAL.html#option--reference-doc

You should at least try to to adapt a template for the Table style and see how it effects the output. 

Though it could be a limitation of what Pandoc allows for tweaking powerpoint. 

If you need advanced styling you could consider using some specific tools from computation language that knows how to output some content for Office Document. 

[flextable](https://ardata-fr.github.io/flextable-book/) R package is one of them, with help of the [officeverse](https://ardata-fr.github.io/officeverse/). It should be compatible with Quarto embedding

[**gt**](https://gt.rstudio.com/) is another option, that knows how to output to Docx, so maybe PPTX too, or you could generate complex layout tables, and export it to PNG to embed in PPTX. 

Python ecosystem could also have some options. 

Hope it helps
